TSC-Charter.md

```diff
@@ -53,10 +53,11 @@ The TSC may, at its discretion, invite any number of non-voting observers to
 participate in the public portion of TSC discussions and meetings.
 
 The TSC shall meet regularly using tools that enable participation by the
-community (e.g. weekly on a Google Hangout On Air, or through any other
+community (e.g. weekly on a Zoom call, or through any other
 appropriate means selected by the TSC). The meeting shall be directed by
-the TSC Chairperson. Responsibility for directing individual meetings may be
-delegated by the TSC Chairperson to any other TSC member. Minutes or an
+the TSC Chairperson or vice Chairperson. Responsibility for directing
+individual meetings may be delegated by the TSC Chairperson or vice
+Chairperson to any other TSC member. Minutes or an
```

Comment:
I don't think we need this change. It already says "Responsibility for directing individual meetings may be delegated by the TSC Chairperson to any other TSC member." So the TSC can choose to delegate a vice chair, and the chair can choose to delegate to the vice chair for any meeting they can't make. If we add the Vice Chair to the Charter, we're tying our hands that we always need to have a Vice Chair, even if we try this and find out it doesn't work.